plugins/com.google.cloud.tools.eclipse.appengine.login/.classpath

```diff
@@ -4,10 +4,12 @@
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="src" path="generated-sources"/>
+	<classpathentry exported="true" kind="lib" path="lib/com.google.cloud.tools.ide.login-0.0.0-SNAPSHOT.jar"/>
```

Comment:
I thought this was a bundle already…?  We should be able to bring it in like we do app-tools-lib